size_test.go

```diff
@@ -93,13 +93,15 @@ func TestFromHumanSize(t *testing.T) {
 	assertSuccessEquals(t, 32*TB, FromHumanSize, "32Tb")
 	assertSuccessEquals(t, 32*PB, FromHumanSize, "32Pb")
 
+	assertSuccessEquals(t, 32.5*KB, FromHumanSize, "32.5kB")
+	assertSuccessEquals(t, 32.5*KB, FromHumanSize, "32.5 kB")
+	assertSuccessEquals(t, 32, FromHumanSize, "32.5 B")
```

Comment:
`FromHumanSize` will always return `int64` instead of `float64`, it will truncate `float64` into `int64`. That's because in my opinion, `32.5 B` isn't meaningful, user don't need to care `bit`, unit as `Byte` is already good enough.

I was thinking to produce error for `32.5 B` and only support `32.5 KB` or `MB` etc, but it will make regexp complicated, so I didn't do that.